packages/hierarchy-builder/src/hierarchy-builder/HierarchyProvider.ts

```diff
@@ -80,8 +103,9 @@ export class HierarchyProvider {
         );
       }),
       concatAll(),
+      concatMap(async (node) => applyLabelsFormatting(node, this._metadataProvider, this._valuesFormatter)),
```

Comment:
Shouldn't this be `concatMap((node) => from(applyLabelsFormatting(node, this._metadataProvider, this._valuesFormatter)))`?

Reply:
using `from` is not necessary - `concatMap` can take a promise as input too